README.rdoc

```diff
@@ -168,9 +168,16 @@ Download and install lighttpd:
 
 Installing the FCGI libraries:
 
+If you use Homebrew on Mac:
```

Comment:
I think Homebrew is available outside of just Mac these days. 🤷‍♂️

Reply:
Oh! 😄